arduino/lidar_control.ino

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
This is the basis for the lidarTilt function. Keeping it here (and commented out) for reference, and until `arduino.ino` has been tested to be able to tilt the LIDAR as expected.